src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/get_interior_vehicle_data_request.cc

```diff
@@ -117,6 +118,20 @@ void GetInteriorVehicleDataRequest::ProcessResponseToMobileFromCache(
   }
 }
 
+bool GetInteriorVehicleDataRequest::CheckRateLimits() {
+  LOG4CXX_AUTO_TRACE(logger_);
+  const uint32_t current_requests_amount =
+      interior_data_cache_.GetCurrentAmountOfRequests();
+  LOG4CXX_DEBUG(logger_,
+                "Current amount of requests in the same time frame is: "
+                    << current_requests_amount);
+  if (current_requests_amount < interior_data_cache_.GetMaxRequestCount()) {
```

Comment:
@ZhdanovP max request count GetInteriorData can take from ini file, not from cache. It is not clear that cach contains max amount of requests just fo giving it to GetINteriorVD